tola/activity_proxy.py

```diff
@@ -49,9 +49,26 @@ def get_by_url(url):
     headers = _get_headers()
     response = requests.get(url, headers=headers)
     content = {}
-
     if response.status_code == 200:
         content = json.loads(response.content)
     else:
         logger.warn('{}: {}'.format(response.status_code, response.content))
     return content
+
+
+def get_workflowlevel1s(user):
+    try:
+        params = {
+            'workflow_user__tola_user_uuid': user.tola_user.tola_user_uuid,
+            'nested_models': True
+        }
+        wfteams = get_workflowteams(**params)
+        wfl1_uuids = []
+        for wfteam in wfteams:
+            if wfteam['workflowlevel1']:
+                wfl1 = wfteam['workflowlevel1']
+                if wfl1:
+                    wfl1_uuids.append(wfl1['level1_uuid'])
+        return wfl1_uuids
+    except Exception:
```

Comment:
You should be more explicit with the exception you want to catch. Pokemon catch'em all is not a good approach.